hack/docker-builder/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fedora:27
```

Comment:
(++code below) I find this approach a bit too `packed`. Having separate container dedicated for rsync and building scales better to the future - different builds (such as generator) can be handled by their own container. Although not visible now, the dockerfile could eventually grow to un-manageable size.

Reply:
I totally understand what you mean and I agree. As you mentioned, let's do that in the future.